sourcegraph/handbook
# Move local SVGs to public folder and update MD tags

Next.js [recommends](https://nextjs.org/docs/basic-features/static-file-serving) serving static content from the `public` directory, so let's move our local SVGs there.

This PR should also:
- Update `object` elements in MD files to be `img` elements.
- Update documentation relating to SVGs.

It will close #2657 

## To Test
- Check out the following pages, and ensure that everything looks good with SVGs loading: `company-info-and-process/communication/asynchronous-communication`, `departments/marketing/brand/brand_guidelines/brand-expression`, `departments/marketing/brand/brand_guidelines/color`, `departments/marketing/brand/brand_guidelines`, `departments/marketing/brand/brand_guidelines/logo`, `departments/people-ops/process/impact-reviews`, `departments/product-engineering/engineering/code-graph/code-insights/processes`.
- Ensure that the new directions make sense at `handbook/editing/handbook-images-video`
- Here are some of the queries I used to test for [`object` elements](https://sourcegraph.com/search?q=context:global+repo:%5Egithub%5C.com/sourcegraph/handbook%24+object+lang:Markdown+&patternType=literal) and stray references to the [`logo` SVG directory](https://sourcegraph.com/search?q=context:global+repo:%5Egithub%5C.com/sourcegraph/handbook%24+logo&patternType=literal) — let me know if I've missed something!
